docs/source/cn/array_ops.py

```diff
@@ -732,3 +732,198 @@
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+
+        >>> a = flow.randn(1, 2, 3, 4, 5)
+        >>> flow.numel(a)
+        120
+        >>> a = flow.zeros(4,4)
+        >>> flow.numel(a)
+        16
+    """
+)
+
+reset_docstr(
+    oneflow.gather_nd,
+    r"""
+    oneflow.gather_nd(input, index) -> Tensor
+    此接口与TensorFlow一致。文档参考自：
+    https://www.tensorflow.org/api_docs/python/tf/gather_nd
+
+
+    该算子将来自 `input` 的切片汇聚成一个新的张量，由 `index` 定义新张量的形状。
+    .. math::
+
+        output[i_{0},i_{1},...,i_{K-2}] = input[index(i_{0},i_{1},...,i_{K-2})]
+
+
+    参数：
+        - **input**: 输入张量
+        - **index**: 切片索引
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> input = flow.tensor([[1, 2,3], [4, 5,6],[7,8,9]], dtype=flow.float)
+        >>> index_1 = flow.tensor([[0], [2]], dtype=flow.int)
+        >>> out_1 = flow.gather_nd(input,index_1)
+        >>> print(out_1.shape)
+        oneflow.Size([2, 3])
+        >>> out_1
+        tensor([[1., 2., 3.],
+                [7., 8., 9.]], dtype=oneflow.float32)
+        >>> index_2 = flow.tensor([[0,2], [2,1]], dtype=flow.int)
+        >>> out_2 = flow.gather_nd(input,index_2)
+        >>> out_2
+        tensor([3., 8.], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.grad_enable,
+    r"""
+    启用梯度计算的上下文管理模式。
+
+    如果其被 no_grad 禁用，则在调用时启用梯度计算。
+
+    此上下文管理模式位于本地线程；不会影响其他线程的计算。
+
+    同时可以作为一种修饰模式。（请确保使用括号实例化）
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> x = flow.ones(2, 3, requires_grad=True)
+        >>> with flow.no_grad():
+        ...     with flow.grad_enable():
+        ...         y = x * x
+        >>> y.requires_grad
+        True
+        >>> @flow.grad_enable()
+        ... def no_grad_func(x):
+        ...     return x * x
+        >>> with flow.no_grad():
+        ...     y = no_grad_func(x)
+        >>> y.requires_grad
+        True
+    """
+)
+
+reset_docstr(
+    oneflow.inference_mode,
+    r"""
+    用于启用或禁用 inference mode 的上下文管理。
+
+    Inference mode 是一个新的上下文管理模式，类似于 no_grad ，可以在你确定你的运算将与 autograd 没有关联时，可以使用这个模式。因为禁用了路径追踪和版本计数堆，在此模式下运行的代码将获得更好的性能。
+
+    此上下文管理模式位于本地线程；不会影响其他线程的计算。
+
+    同时可以作为一种修饰模式。（请确保使用括号实例化）
+
+    参数：
+        - **mode** (bool): 标记启用或禁用 inference mode (默认：True)
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> x = flow.ones(2, 3, requires_grad=True)
+        >>> with flow.inference_mode():
+        ...     y = x * x
+        >>> y.requires_grad
+        False
+        >>> @flow.inference_mode()
+        ... def no_grad_func(x):
+        ...     return x * x
+        >>> y = no_grad_func(x)
+        >>> y.requires_grad
+        False
+    """
+)
+
+reset_docstr(
+    oneflow.save,
+    r"""保存一个对象到一个路径。
+
+    参数：
+        - **obj**: 被保存的对象
+        - **path** (str): 对象被保存的路径
+        - **consistent_dst_rank** (int, 可选): 用于保存全局张量的地点秩。被指定时，对于所有张量，只有秩 == consistent_src_rank 的进程被保存，而其他的进程不会进行任何磁盘存取。
```

Comment:
```suggestion
        - **consistent_dst_rank** (int, 可选): 用于保存全局张量的地点秩。被指定时，对于所有张量，只有 rank == consistent_src_rank 的进程被保存，而其他的进程不会进行任何磁盘存取。
```

这里的 rank 不用翻译了，因为它不是线性代数里的秩，其实就是编号的意思。这个含义，在集合通信里已经比较通用

Reply:
已解决